etc/env/example.env

```diff
@@ -2,7 +2,7 @@
 # - - -
 
 # site options
-CANDIG_MODULES=weavescope portainer consul traefik logging monitoring minio drs-server htsget-server chord-metadata datasets cnv-service rnaget candig-server federation-service cancogen-dashboard #wes-server jupyter igv authentication authorization
+CANDIG_MODULES=candig-data-portal weavescope portainer consul traefik logging monitoring minio drs-server htsget-server chord-metadata datasets cnv-service rnaget candig-server federation-service cancogen-dashboard #wes-server jupyter igv authentication authorization
```

Comment:
Order of CANDIG_MODULES is important. In the future make sure you move front-end services near the end of the list as they should usually deploy after backend services